src/main/java/org/cactoos/text/Capitalized.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Func;
+import org.cactoos.Text;
+import org.cactoos.scalar.Ternary;
+
+/**
+ * Text in capitalized case, changed the first character to title case, no other
+ * characters are changed.
+ *
+ * @since 0.46
+ */
+public final class Capitalized extends TextEnvelope {
+
+    /**
+     * Ctor.
+     *
+     * @param text The text
+     */
+    public Capitalized(final String text) {
+        this(new TextOf(text));
+    }
+
+    /**
+     * Ctor.
+     *
+     * @param text The text
+     */
+    public Capitalized(final Text text) {
+        super(
+            new TextOf(
+                () -> {
+                    final Func<String, String> capitalize = cap -> {
```

Comment:
@kokodyn 
I believe it should be more like:
```
super(new Ternary<>(new IsBlank(text), () -> text, new Ternary<>(new StartsWith(...), ...);
```
Also, `Func<String, String>` should be a `Text` but without extra checks

Reply:
Ok. Not sure what does it mean "without extra checks". Please look at my proposition in newest commit.

Reply:
@kokodyn It looks complicated. 
Why it cannot be like that?
```
                new Ternary<Text>(
                    new IsBlank(text),
                    text,
                    new Joined(
                        "",
                        new Upper(new Sub(text, 0, 1)),
                        new Sub(text, 1)
                    )
                )
```

May be imperative approach was better here.

Reply:
@andreoss While working on that PR I realized that there is a difference between Upper and TitleCase:
https://stackoverflow.com/questions/31770995/difference-between-uppercase-and-titlecase

Reply:
@kokodyn I agree with @andreoss, why not use the much simpler example he proposes?
- no need for testing if the first character is already title case or not, let's just do it as in @andreoss example above
- let's use Upper for now
- let's add a `@todo` to the class so that we add `Titled` and replace `Upper` in this class.

Reply:
@victornoel @andreoss 
I will simplify logic as you suggested.
I created new PR to add Titled, could you look at it, please?
 I think firstly we can merge Titled and use it in Capitalized PR, ok?

Reply:
@kokodyn that's acceptable, but please take note that the purpose of cactoos is to be a sandbox area for experimenting with Zerocracy, on top of implementing this library. Hence in the future it would be best to follow the proper workflow which advocates for using `@todo`s as a way to decompose work. Once from time to time, it is ok to not follow this, but this should remain the general nominal way of working here.

Reply:
@victornoel 
i added @TODO comment to with information what should be replaced in future.
I would like to have it proper working(and later replaced using static titled by "EO Titled" ) instead of using Lower.
As I said before this is tricky difference beetwen Lower and Titled.
For instance:
Title case: ǅ ->https://www.compart.com/en/unicode/U+01C5
Lower case: ǆ -> https://www.compart.com/en/unicode/U+01C6
Upper case: Ǆ -> https://www.compart.com/en/unicode/U+01C4

However, if you are sure that Lower should be used please let me know.

Reply:
@kokodyn I thought we were talking about `Upper` vs `Titled`, no? What I propose is to tackle this issue incrementally, so first we make it work with Upper because it's good enough, and then with the todo we explore how to have Capitalized use titled characters.

Reply:
@kokodyn anywa, see my last comment about the current code, maybe we can just use title case as-is right now without introducing any new class.

Reply:
Yes we can use tittle case without introducing any more class.